doc/api/async_hooks.md

```diff
@@ -538,9 +537,8 @@ own resources.
 
 The `init` hook will trigger when an `AsyncResource` is instantiated.
 
-*Note*: It is important that `before`/`after` calls are unwound
-in the same order they are called. Otherwise an unrecoverable exception
-will occur and the process will abort.
+`before` and `after` calls are unwound in the same order they are called.
```

Comment:
I think this should be `..calls must be unwound...` since it's a note for the implementers.